src/app/theme/SplashScreenService.ts

```diff
@@ -2,19 +2,31 @@ import RNBootSplash, { VisibilityStatus } from 'react-native-bootsplash'
 
 import { flagshipUIEventHandler, flagshipUIEvents } from '/app/view/FlagshipUI'
 import { devlog } from '/core/tools/env'
+import { logToSentry } from '/libs/monitoring/Sentry'
+import config from '/app/theme/config.json'
 
 export const splashScreens = {
   LOCK_SCREEN: 'LOCK_SCREEN',
   SECURE_BACKGROUND: 'secure_background' // this mirrors native declaration
 } as const
+
 type SplashScreenEnumKeys = keyof typeof splashScreens
 export type SplashScreenEnum = (typeof splashScreens)[SplashScreenEnumKeys]
 
-export class SplashScreenService {
-  show = showSplashScreen
-  hide = hideSplashScreen
+let autoHideTimer: NodeJS.Timeout | null = null
+let autoHideDuration = config.autoHideDuration // Default timeout duration in milliseconds
```

Comment:
Why is it outside in a global value ? It is not possible to pass it as a param of `showSplashScreen` with a default value ?

Reply:
I preferred to separate what I considered as plain config from the actual code, so the timeout value is implicit/declarative and at the same time clearly visible in the config file